tools/webhooks.md

```diff
@@ -8,6 +8,7 @@ other notifications channels:
 -   Every occurrence (`occurrence`)
 -   Item reactivated (`reactivated_item`)
 -   10^nth occurrence (`exp_repeat_item`)
+-   High occurrence rate (`item_velocity`)
```

Comment:
Maybe we should sort these? (alphabetically). same for the EVENT_NAME list below